jsmvc/index.html

```diff
@@ -387,7 +387,7 @@ <h1 class="upper">JavaScript MVC</h1>
 			</div>			
 		</div>
 
-		<script src="../common/js/head.min.js"></script>
+		<script src="../common/lib/js/head.min.js"></script>
```

Comment:
what's the difference between `lib/js/` and regular `js/`?
I could see:
* `js/` being the custom files
* `lib/js` being the libraries
but then shouldn't `reveal.min.js` be in the `lib/js/` folder as well?